### PR TITLE
fix: Broken links in README.md

### PR DESCRIPTION
**Pull request by Agnik7**
### Description

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

Description: This PR fixes the Issue of the broken links in the README, which have been fixed and aligned with the AWS blog.

- Relevant Issues : (compulsory) #124 
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [x] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
 
 ## ➤ Table of Contents
```
Comment (namitad) [inline README.md:29]: you can remove the `➤` from here as well for uniformity

File: README.md [hunk 2]
```diff
@@ -28,19 +28,19 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
...
-- * [➤ How it Works](#-how-it-works)
-- * [➤ Contributing to Trapheus](#-contributing-to-trapheus)
-- * [➤ Contributors](#-contributors)
+- * [Pre-Requisites](#pre-requisites )
+- * [Parameters](#parameters)
+- * [Instructions](#instructions)
+- * [Execution](#execution)
+- * [How it Works](#how-it-works)
+- * [Contributing to Trapheus](#contributing-to-trapheus)
+- * [Contributors](#contributors)
```
Comment (namitad) [inline README.md:37]: lets retain the `➤` in the table of contents

**Review by namitad — COMMENTED** [review]
@Agnik7 have added some minor comments. please address. will merge post that.

File: README.md (hunk 1 above)
Comment (Agnik7) [reply README.md:29]: > you can remove the `➤` from here as well for uniformity

Ok, so should I remove this `➤` or retain it? The broken link problem is actually occurring due to this character.

**Comment by Agnik7** [comment]
@namitad  So the `➤` character should be present in the Table of contents right?

**Comment by namitad** [comment]
@Agnik7 you can remove it from the `Table of Contents` header and retain it for the list within

**Comment by Agnik7** [comment]
@namitad  Doing it right now

**Comment by Agnik7** [comment]
@namitad please check.

**Comment by Agnik7** [comment]
@namitad I have a small request. If possible, could you kindly add the hacktoberfest topic in this repository? Otherwise this PR will not be counter for Hacktoberfest. Apologies for any inconveniences caused.

**Review by namitad — APPROVED** [review]

**Comment by namitad** [comment]
@Agnik7 Thank you for your first contribution to Trapheus! I have added the required hacktoberfest labels.
 Do add a star for our github repo. looking forward to more contributions!

**Pull request merged**